src/main/java/seedu/address/logic/commands/ExportCertCommand.java

```diff
@@ -39,8 +39,7 @@
             + "Example: " + COMMAND_WORD + " 1";
 
     public static final String MESSAGE_ARGUMENTS = "Index: %1$d";
-    public static final String MESSAGE_EXPORT_CERT_SUCCESS = "Certificate exported for volunteer at %1$d "
-            + "to your Desktop.";
+    public static final String MESSAGE_EXPORT_CERT_SUCCESS = "Certificate exported for volunteer at %1$d to ";
     public static final String MESSAGE_EXPORT_FAILED = "Certificate export failed, please try again.";
     public static final String PDF_SAVE_PATH = System.getProperty("user.dir") + "/Volunteer Certs/";
```

Comment:
Also, can maybe put the directory path in UserPrefs.java and preferences.json so that the user

Reply:
Added to #175